src/pages/Loans/LoansOverview/components/LendAssetsTable/LendAssetsTable.tsx

```diff
@@ -50,14 +49,16 @@ const LendAssetsTable = ({ assets, onRowAction, disabledKeys }: LendAssetsTableP
       Object.values(assets).map(({ lendApy, lendReward, currency, totalLiquidity }) => {
         const asset = <AssetCell currency={currency.ticker} />;
 
-        const rewardsApy = getSubsidyRewardApy(currency, lendReward, prices);
-        const formattedRewardsApy = lendReward
-          ? `${lendReward?.currency.ticker}: ${formatPercentage(rewardsApy || 0, {
-              maximumFractionDigits: 2
-            })}`
-          : undefined;
-
-        const apy = <ApyCell apy={lendApy} amount={formattedRewardsApy} />;
+        const apy = (
+          <ApyCell
+            assetApy={lendApy}
+            assetCurrency={currency}
+            rewards={lendReward}
+            prices={prices}
+            // TODO: temporary until we find why row click is being ignored
+            onClick={() => onRowAction?.(currency.ticker as Key)}
```

Comment:
is this a problem on a table component level?

Reply:
I dont think so, but its not a blocker